### PR TITLE
Release automatization

### PR DESCRIPTION
**Pull request by solita-alperttiti**
This pull request resolves COVID-380.

### CODE REVIEW

File: publish-token/pom.xml [hunk 1]
```diff
@@ -4,17 +4,17 @@
 
 	<modelVersion>4.0.0</modelVersion>
 
+    <groupId>fi.thl.covid19</groupId>
```
Comment (jyrkija) [inline publish-token/pom.xml:7]: different indentation?

File: exposure-notification/pom.xml [hunk 2]
```diff
@@ -4,17 +4,17 @@
 
 	<modelVersion>4.0.0</modelVersion>
 
+    <groupId>fi.thl.covid19</groupId>
```
Comment (jyrkija) [inline exposure-notification/pom.xml:7]: Indentation here too. Is this a github-thing or real difference?

File: create_release_branch.sh [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+git fetch --all --tags --prune
+git checkout trunk
+git pull --rebase
+
+CURRENT_VERSION=$(grep -oPm1 "(?<=<version>)[^<]+" pom.xml)
+
+git checkout -b release/$CURRENT_VERSION
+
+while true; do
+    read -rp "Do you want to push new release branch to remote?" yn
```
Comment (jyrkija) [inline create_release_branch.sh]: Might be helpful to include the version(s) in this query text. Same thing below.

**Review by jyrkija — CHANGES_REQUESTED** [review]

File: publish-token/pom.xml (hunk 1 above)
Comment (solita-alperttiti) [reply publish-token/pom.xml:7]: Good question. It seems to be fine in my end.

File: exposure-notification/pom.xml (hunk 2 above)
Comment (solita-alperttiti) [reply exposure-notification/pom.xml:7]: Good question. It seems to be fine in my end.

Comment (solita-alperttiti) [reply exposure-notification/pom.xml:7]: No, there is tabs... I'll fix that

File: create_release_branch.sh (hunk 3 above)
Comment (solita-alperttiti) [reply create_release_branch.sh]: Yep, this is a good one. Added.

**Review by jyrkija — APPROVED** [review]

**Pull request merged**